server/api/user/user.controller.js

```diff
@@ -29,6 +32,40 @@ function respondWith(res, statusCode) {
 }
 
 /**
+ * Verifies that a user's email has been confirmed
+ */
+function verifyUser(token, callback){
+    verificationTokenModel.findOne({token: token})
+      .then((doc) => {
+        if (doc == null) return callback("[Error] No token found");
```

Comment:
Confusing error message. Maybe "Error: This token is invalid". It is also not used anywhere?